tests/testJointcal.py

```diff
@@ -62,11 +62,16 @@ class JointcalTest(lsst.utils.tests.TestCase):
     @classmethod
     def setUpClass(cls):
         """Can only do the eups setup once, so have to do it here only."""
+
         # NOTE: this may be cheating? How else should I do this?
-        eups.setup('astrometry_net_data', productRoot=os.path.join(os.environ['JOINTCAL_DIR'], 'tests/data/'))
+        jointcal_reference_dir = os.path.join(os.environ['VALIDATION_DATA_JOINTCAL_DIR'],
+                                              'twinkles1_and_index')
+        eups.setup('astrometry_net_data', productRoot=jointcal_reference_dir)
```

Comment:
This worries me. Is it needed? I would have thought that having validation_data_jointcal setup would suffice.